### PR TITLE
Upgrade JS deps.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,67 +45,66 @@
...
-    "@rollup/plugin-node-resolve": "^13.0.5",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-node-resolve": "^13.0.6",
     "@tommoor/remove-markdown": "^0.3.2",
     "@types/markdown-it": "^12.2.3",
-    "@typescript-eslint/parser": "^4.33.0",
-    "@yarnpkg/sdks": "^2.4.2-0",
+    "@typescript-eslint/parser": "^5.6.0",
+    "@yarnpkg/sdks": "^2.5.1-rc.5",
     "accoutrement": "^3.0.1",
```
Comment (jgerigmeyer) [inline package.json:60]: @mirisuzanne I think we could also upgrade Accoutrement (and Sass) in this PR?

Comment (mirisuzanne) [reply package.json:59]: yes please 👍🏼

**Review by jerivas — APPROVED** [review]
Not sure if I should be looking for anything in particular, but the diff here and on the built files look good to me 👍

File: package.json (hunk 1 above)
Comment (jgerigmeyer) [reply package.json:59]: Doesn't need to hold up this PR.

**Pull request merged**